code/modules/projectiles/guns/projectile/sniper.dm

```diff
@@ -188,3 +181,24 @@
 		breakthings = TRUE
 	else if(damage > 25)
 		icon_state = "gauss"
+
+// Savior of Lowpop rounds
+
+/obj/item/weapon/gun/projectile/sniper_rifle/emergency
+	name = "emergency self-defense rifle"
+	desc = "Nanotrasen deploys these cheap mass-produced firearms wherever undermanned stations are at risk"
+	icon_state = "moistnugget"
```

Comment:
is THIS why you did the above? because that's a bad solution.
you need to write this:

```
/obj/item/weapon/gun/projectile/sniper_rifle/emergency/update_icon()
    return
```

instead of what you did above.